releasenotes.txt

```diff
@@ -342,6 +369,14 @@ Release notes - NHibernate - Version 5.2.0
 As part of releasing 5.2.0, a misnamed setting in 5.0.0 release notes has been fixed:
 transaction.use_connection_on_system_events correct name is transaction.use_connection_on_system_prepare
 
+Build 5.1.6
```

Comment:
I have included the backports release notes (5.1.6 and 5.0.8) in the async regen commit.

Reply:
It seems that async regen should be embedded to the merge commit itself. And release notes to be on own commit.

Reply:
I am not used to edit files not having any conflicts in a merge commit. I was even not thinking this possible. I will check.
(Or maybe add in my local 5.2.x a commit for this prior to the merge, then drop it from my local 5.2.x ? Looks too hackish, and even worst.)

Reply:
You edit files before merge (there was a conflict or --no-commit option used) and then you `git add <path>` the manually edited files; then `git commit`.

Reply:
Ok, thanks for the hint.